spec/requests/api/v1/customer_request_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'rails_helper'
+
+describe "customer requests" do 
+  it "returns data for all customers" do 
+    create_list(:customer,3)
+
+    get "/api/v1/customers"
+    customers = JSON.parse(response.body)
+
+    expect(response).to be_successful 
+    expect(customers.first).to have_key("first_name")
```

Comment:
We should add some more here to test for all the data that we want for a customer- I think last_name too?